scripts/parse_prophecy.py

```diff
@@ -97,18 +121,21 @@ async def main(
     manifests: List[str],
     sample_metadata_project: str,
     search_locations: List[str],
-    confirm=False,
+    confirm=True,
     dry_run=False,
 ):
     """Run script from CLI arguments"""
     parser = ProphecyParser(
         sample_metadata_project=sample_metadata_project,
         search_locations=search_locations,
-        sample_name_column='Sample/Name',
+        sample_name_column='External ID',
```

Comment:
Remind me, what did we agree with regards to sample IDs? I recall that we discussed using `participantid` (e.g. `658`) instead of `Specimen ID`=`External ID` (e.g. `6580308`) as the DB participant IDs, and probably keep those `220208_FD02700719` as DB external sample IDs. But I didn't take notes and forgot what we ended up with :(

Reply:
Ah, perhaps `External ID`(`6580308`) should match external sample IDs in the DB, and `658` should be the participant ID

Reply:
Yes, you're right! External ID = Specimen ID which is the Sample ID.  I've fixed it now. `658` would be the Participant ID.  Fixed now.